lib/node_modules/@stdlib/constants/float32/max-safe-nth-fibonacci/lib/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
This is not acceptable. Please do not disable linting like this. We have linting enabled for a reason. Disabling wholesale increases the risk of bugs.